dist/api_data.py

```diff
@@ -1691,7 +1683,7 @@ def __ge__(self, other):
         API call. There is no guarantee any operation will not take longer than this amount of time.
         If WiredTiger notices the limit has been exceeded, an operation may return a WT_ROLLBACK
         error. Default is to have no limit''',
-        min=1),
```

Comment:
I see that we want to allow a zero value to be passed but how does that tie in with making the minimum 1?